commands/chain.go

```diff
@@ -40,37 +45,177 @@ var ChainCmd = &cli.Command{
 		ChainListCmd,
 		ChainSetHeadCmd,
 		ChainActorCodesCmd,
+		ChainActorMethodsCmd,
 		ChainStateInspect,
 		ChainStateCompute,
 		ChainStateComputeRange,
 	},
 }
 
+type chainActorOpts struct {
+	persist bool
+	config  string
+	storage string
+}
+
+var chainActorFlags chainActorOpts
+
+var persistFlag = &cli.BoolFlag{
```

Comment:
I think the presence of the `storage` flag should be enough to indicate persistence is desired by the users. Can we drop this flag and just assert if the `storage` flag is set?

Reply:
sounds good